metabase/metabase
# Show inactive tables in database & table pickers in the query builder

Epic: https://github.com/metabase/metabase/issues/18675

1. For databases - if the initial sync for all tables hasn't finished
2. For tables - if the initial sync for this table hasn't finished

DatabaseSchemaPicker:
<img width="328" alt="Screen Shot 2021-10-27 at 10 28 06 pm" src="https://user-images.githubusercontent.com/8542534/139134476-d1559f3a-51cc-4abd-9979-25f499999def.png">

TablePicker:
<img width="318" alt="Screen Shot 2021-10-27 at 10 31 42 pm" src="https://user-images.githubusercontent.com/8542534/139134482-7750eebc-d72a-4ddf-be67-762baa1cddbb.png">

How to test:
As the BE for this PR is not ready we would need to rely on mocked data:
1. Set `initial_sync` to false https://github.com/metabase/metabase/blob/c0b5c697d1c5c814ec35a50ed2696617ea172d4a/frontend/src/metabase/schema.js#L19
2. Set `initial_sync` to false https://github.com/metabase/metabase/blob/c0b5c697d1c5c814ec35a50ed2696617ea172d4a/frontend/src/metabase/schema.js#L53
3. Add a question -> Simple question. 
4. If your instance has only one database, click on the back icon to navigate to Database picker.
5. Check that databases have spinners but it's possible to select them.
6. Check that tables are disabled and it's not possible to select them.
7. Revert `initial_sync` changes and check that everything works as usual.
